pallets/asset-manager/Cargo.toml

```diff
@@ -8,25 +8,25 @@ license    = 'GPL-3.0'
 repository = 'https://github.com/Manta-Network/Manta/'
 
 [dependencies]
-codec = { package = "parity-scale-codec", version = '3.0.0', default-features = false }
-scale-info = { version = "2.0.0", default-features = false, features = ["derive"] }
-sp-runtime = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.18", default-features = false }
-frame-support = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.18", default-features = false }
-frame-system = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.18", default-features = false }
-sp-std = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.18", default-features = false }
-frame-benchmarking = { git = 'https://github.com/paritytech/substrate.git', branch = "polkadot-v0.9.18", default-features = false, optional = true }
-xcm = { git = 'https://github.com/paritytech/polkadot.git', default-features = false, branch = "release-v0.9.18" }
+codec = { package = "parity-scale-codec", version = '3.1.2', default-features = false }
+scale-info = { version = "2.1.2", default-features = false, features = ["derive"] }
+sp-runtime = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.22", default-features = false }
+frame-support = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.22", default-features = false }
+frame-system = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.22", default-features = false }
+sp-std = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.22", default-features = false }
+frame-benchmarking = { git = 'https://github.com/paritytech/substrate.git', branch = "polkadot-v0.9.22", default-features = false, optional = true }
+xcm = { git = 'https://github.com/paritytech/polkadot.git', default-features = false, branch = "release-v0.9.22" }
 manta-primitives = { path = "../../primitives", default-features = false }
 log = { version = "0.4.0", default-features = false }
 
 # 3rd party dependencies
-orml-traits = { git = "https://github.com/manta-network/open-runtime-module-library.git", default-features = false, rev="7e2f985" }
+orml-traits = { git = "https://github.com/manta-network/open-runtime-module-library.git", default-features = false, rev="cc26ffa" }
```

Comment:
Why don't we use the release branches instead of the concrete commits ?
They use the same same scheme as Parity repos.

Reply:
Oh wait why are we depending on our own fork ?
We don't have any custom code.

Reply:
I guess that's fine, but it's inconsistent with how we deal with all the rest of the upstream repos.